<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBox::tab {
    background-color: rgb(75, 75, 75);
    color: white;
}

QListView {
   background-color: rgba(255, 255, 255, 255); 
}

QMainWindow::separator { 
  width: 1px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="_7">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="plotWidgetLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="PlotWidget" name="plotWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>455</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Modules</string>
    </property>
    <widget class="QMenu" name="menuModulesQuanty">
     <property name="title">
      <string>Quanty</string>
     </property>
     <addaction name="quantyModuleHide"/>
     <addaction name="separator"/>
     <addaction name="quantySaveInput"/>
     <addaction name="quantySaveAsInput"/>
     <addaction name="separator"/>
     <addaction name="quantyRunCalculation"/>
    </widget>
    <addaction name="menuModulesQuanty"/>
   </widget>
   <addaction name="menuModules"/>
  </widget>
  <widget class="QDockWidget" name="quantyDockWidget">
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>350</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Quanty</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="quantyDockWidgetContents">
    <layout class="QVBoxLayout" name="_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBox" name="quantyToolBox">
       <widget class="QWidget" name="generalSetupPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>240</width>
          <height>534</height>
         </rect>
        </property>
        <attribute name="label">
         <string>General Setup</string>
        </attribute>
        <layout class="QVBoxLayout" name="_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="elementAndSymmetryGroupBox">
           <property name="title">
            <string>Element and Symmetry</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="ComboBox" name="elementComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ComboBox" name="chargeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ComboBox" name="symmetryComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="theoreticalModelGroupBox">
           <property name="title">
            <string>Theoretical Model</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="ComboBox" name="theoreticalModelComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="experimentGroupBox">
           <property name="title">
            <string>Experiment</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="ComboBox" name="experimentComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ComboBox" name="edgeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="broadeningGroupBox">
           <property name="title">
            <string>Broadening FWHM (eV)</string>
           </property>
           <layout class="QVBoxLayout" name="_6">
            <item>
             <layout class="QHBoxLayout" name="broadeningGaussianHorizontalLayout">
              <item>
               <widget class="QLabel" name="broadeningGaussianLabel">
                <property name="text">
                 <string>Gaussian</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="broadeningGaussianDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.400000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="broadeningLorentzianHorizontalLayout">
              <item>
               <widget class="QLabel" name="broadeningLorentzianLabel">
                <property name="text">
                 <string>Lorenzian</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="broadeningLorentzianDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="minimum">
                 <double>0.200000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.400000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="temperatureGroupBox">
           <property name="title">
            <string>Temperature (K)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="temperatureHorizontalLayout">
              <item>
               <widget class="QLabel" name="temperatureLabel">
                <property name="text">
                 <string>T</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="temperatureDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="magneticFieldGroupBox">
           <property name="title">
            <string>Magnetic Field (T)</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="magneticFieldXHorizontalLayout">
              <item>
               <widget class="QLabel" name="magneticFieldXLabel">
                <property name="text">
                 <string>Bx</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="magneticFieldXDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="magneticFieldYHorizontalLayout">
              <item>
               <widget class="QLabel" name="magneticFieldYLabel">
                <property name="text">
                 <string>By</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="magneticFieldYDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="magneticFieldZHorizontalLayout">
              <item>
               <widget class="QLabel" name="magneticFieldZlabel">
                <property name="text">
                 <string>Bz</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="magneticFieldZDoubleSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
        <zorder>elementAndSymmetryGroupBox</zorder>
        <zorder>experimentGroupBox</zorder>
        <zorder>theoreticalModelGroupBox</zorder>
        <zorder>temperatureGroupBox</zorder>
        <zorder>magneticFieldGroupBox</zorder>
        <zorder>broadeningGroupBox</zorder>
        <zorder>verticalSpacer</zorder>
       </widget>
       <widget class="QWidget" name="hamiltonianSetupPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>161</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Hamiltonian Setup</string>
        </attribute>
        <layout class="QVBoxLayout" name="_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QListView" name="hamiltonianTermsView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>110</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="TreeView" name="hamiltonianParametersView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="resultsPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>79</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Results</string>
        </attribute>
        <layout class="QVBoxLayout" name="_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListView" name="resultsView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="toolTip">
          <string>Save a Quanty input file</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>icons/save.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runPushButton">
         <property name="toolTip">
          <string>Run Quanty</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>icons/play.svg</normalon>
          </iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quantySaveInput">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the Quanty Input File</string>
   </property>
  </action>
  <action name="quantyRunCalculation">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="quantySaveAsInput">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="quantyModuleShow">
   <property name="text">
    <string>Show</string>
   </property>
   <property name="toolTip">
    <string>Show the Quanty Module</string>
   </property>
  </action>
  <action name="quantyModuleHide">
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="toolTip">
    <string>Hide the Quanty Module</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>.views.treeview</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>.widgets.combobox</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>.widgets.plotwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
